packages/cli/src/utils/safe.ts

```diff
@@ -0,0 +1,69 @@
+import { StrongAddress } from '@celo/base'
+import { CeloTransactionObject } from '@celo/connect'
+import { CeloProvider } from '@celo/connect/lib/celo-provider'
+import Safe from '@safe-global/protocol-kit'
+
+export const createSafeFromWeb3 = async (
+  web3: any,
```

Comment:
suggestion. instead of passing web3 pass celoProvider: CeloProvider.

Principle of least need

Reply:
I'd rather leave it as is because the check 

```
if (!(web3.currentProvider instanceof CeloProvider)) {
    throw new Error('Unexpected web3 provider')
  }
```

is crucial for the typing. Otherwise I'd have to either cast or perform this check wherever the safe is created.